src/main/cms/cms.c

```diff
@@ -239,10 +242,18 @@ static uint8_t cmsCursorAbsolute(displayPort_t *instance)
     return currentCtx.cursorRow + currentCtx.page * maxMenuItems;
 }
 
+uint8_t runtimeEntryFlags[CMS_MAX_ROWS] = {0};
```

Comment:
We normally use a space on the inside of curly braces.

Reply:
fixed.